scripts/generate_vcxproj

```diff
@@ -16,6 +16,9 @@ function doit {
   for dir in $dirs ; do
     sources="`(cd $dest/src/$dir; make sources)`"
     for s in $sources ; do
+      if [ "$s" = "goto_instrument_main.cpp" ] && [ "$s" != "${1}_main.cpp" ] ; then
+        continue
```

Comment:
The first conjunct isn't subsumed by the second?

Reply:
We want to `continue` if seeing `goto_instrument_main.cpp` while not actually building `goto-instrument.vcxproj` - which actually tells me I still need to fix this and change the second conjunct to `[ "$1" != "goto-instrument" ]`... will fix.

Reply:
Very minor nit pick on inconsistent use of `${blah}` versus `$blah`?